bliss/encoder/encoder.py

```diff
@@ -1,6 +1,7 @@
 from copy import copy
 from typing import Optional
 
+import numpy as np
```

Comment:
can we use torch instead to avoid an import?

Reply:
Done